mysql-test/t/audit_log_many_connections.test

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
The test case for THDVAR_SET in upstream added its invocation to example storage engine plugin